{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/settings/tests.py

```diff
@@ -0,0 +1,26 @@
+from .base import *
+
+DEBUG = False
+SECRET_KEY = "secret_key"
+
+# ------------- LOGGING -------------
+LOGGING = {}
+
+# ------------- MIDDLEWARES -------------
+MIDDLEWARE = list(filter(lambda x: "DebugToolbarMiddleware" not in x, MIDDLEWARE))
+
+# ------------- PASSWORDS -------------
+PASSWORD_HASHERS = [
+    "django.contrib.auth.hashers.MD5PasswordHasher",
+]
+
+# ------------- DATABASES -------------
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": env("POSTGRES_DB", "{{ cookiecutter.project_name }}"),
+        "USER": env("POSTGRES_USER", "{{ cookiecutter.project_name }}"),
+        "PASSWORD": env("POSTGRES_PASSWORD", "{{ cookiecutter.project_name }}"),
+        "HOST": env("POSTGRES_HOST", "localhost"),
+    }
+}
```

Comment:
`pytest` detects this file as a test file and because of it - fails.
I think that the best solution will be to exclude the entire `settings` directory from searching for test modules.

Reply:
I add path in pyproject.toml in section [tool.pytest.ini_options]